sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/batch/PartitionKeyRangeServerBatchRequest.java

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.implementation.batch;
+
+import com.azure.cosmos.CosmosItemOperation;
+import reactor.core.publisher.Mono;
+
+import java.util.List;
+
+import static com.azure.cosmos.implementation.guava25.base.Preconditions.checkNotNull;
+
+public final class PartitionKeyRangeServerBatchRequest extends ServerBatchRequest {
```

Comment:
still we are passing RxDocumentServiceRequest to the lower part of stack. 

PartitionKeyRangeServerBatchRequest at some point will be translated to a RxDocumentServiceRequest. am I right?

Reply:
Yeah, this is just a container to get values when we are converting to RxDocumentServiceRequest.